package.json

```diff
@@ -31,5 +31,8 @@
     "postcss": "^8.4.31",
     "tailwindcss": "^3.3.5",
     "typescript": "^5.2.2"
+  },
+  "engines" : {
+    "node" : ">=19.0.0 <=20.8.9"
```

Comment:
[Please Change]: This is incorrect per SemVer - with SemVer specifications, we should specify the lower bound as the minimum version supported. Since we're using Node 20, this should be at least `20.0.0`. It's also possible to specify higher versions that way we don't test on (say) `20.8.9` successfully but earlier versions might be missing features or have bugs. Two options:

 - `>= 20.0.0.`, aka any version of Node 20 (Recommended: this is the closest match to our workflow files right now which use `20.x`).
 - `>= 20.8.9`, aka at least the current version of Node which has been tested. This is also fine, but matching our workflows makes more sense.